### PR TITLE
We are looking for a new maintainer

### PR DESCRIPTION
**Pull request by gabebw**

### CODE REVIEW

**Comment by gabebw** [comment]
@mike-burns I trust your judgment on the wording for this.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
-# fake\_braintree, a Braintree fake [![Build Status](https://secure.travis-ci.org/thoughtbot/fake_braintree.svg)](http://travis-ci.org/thoughtbot/fake_braintree)
+# We are looking for a maintainer
+
+We no longer use Braintree but we want this library to live on.
+
+If you're interested, reach out to the current maintainer, Gabe Berke-Williams,
```
Comment (JoelQ) [inline README.md]: Thoughts on being more explicit?

> If you're interested in *taking over this project*, reach out to ...

Comment (gabebw) [inline README.md]: Oh hey yeah.

**Comment by JoelQ** [comment]
LGTM :+1:

**Pull request closed without merging**